Haylea

hayrenguul opened the pull request.

hayrenguul committed:
Add files via upload

hayrenguul committed:
Moved SW1-SW3, changed J1-J3 footprints

Moved SW1-SW3 closer to the checks, changed J1-J3 footprints to vertical footprints

HannahSLee14 reviewed (commented):
It looks better! There are just a couple more edits that are super small. 

- Can you make sure test points TP3 and TP4 are aligned? 

- Can you also make J1-J3 are all aligned and the same distance away from the right edge as J4-J5 are from the left edge, and can you rotate J1-J3 by 180 degrees so that the little yellow dots are facing inward like J4-J5? 

- Finally, can you move C1 closer to the power connector? 

Thanks Haylea - you're doing great!

hayrenguul committed:
Add files via upload

Aligned TP3 and TP4, aligned and rotated J1-J3, moved C1 closer to the power connector.

HannahSLee14 reviewed (commented):
It's looking good so far! I just have a few more suggestions

![image](https://user-images.githubusercontent.com/55327249/92986849-4e504d00-f483-11ea-90cf-ce9835a0bc5a.png)
![image](https://user-images.githubusercontent.com/55327249/92986857-590ae200-f483-11ea-8e0b-52a1d8657779.png)

hayrenguul committed:
Moving components around

On the schematic: Connected pin 1 of J4 to +12V, placed SW2 and SW3 before R6 and R7 (respectively).
On the PCB: Used vias to connect D4 and D5 to +12V, updated the label, made the arrangement of components for the contactor check and precharge check more symmetric, got rid of the extra trace coming out of Q4.

HannahSLee14 reviewed (commented):
Just a few more comments! 
![image](https://user-images.githubusercontent.com/55327249/93001993-f05e4c80-f4f8-11ea-9c50-c66b3e304452.png)

hayrenguul committed:
Aligning components

Aligning the components on the right side of the board to their counterparts, deleting extra traces on the left side of the board.

hayrenguul committed:
Added dimensions

HannahSLee14 reviewed (commented):
It looks good!
